google/oauth2/_client.py

```diff
@@ -214,20 +293,24 @@ def jwt_grant(request, token_uri, assertion):
     """
     body = {"assertion": assertion, "grant_type": _JWT_GRANT_TYPE}
 
-    response_data = _token_endpoint_request(request, token_uri, body)
+    response_data = _token_endpoint_request(
+        request, token_uri, body, can_retry=can_retry
+    )
 
     try:
         access_token = response_data["access_token"]
     except KeyError as caught_exc:
-        new_exc = exceptions.RefreshError("No access token in response.", response_data)
+        new_exc = exceptions.RefreshError(
+            "No access token in response.", response_data, retryable=True
```

Comment:
I understand it is existing behavior.. but i wonder how it works for example when refresh fail due to reauth? 
How this does not retry indefinitely?

Reply:
Good point, I changed this to False